tests/test-cases/Group23-VIC-Machine-Service/23-02-VCH-List.robot

```diff
@@ -27,7 +27,7 @@ Setup
     Install VIC Appliance To Test Server
 
 Teardown
-    Run Keyword And Continue On Failure  Cleanup VIC Appliance On Test Server
+    Run Keyword And Ignore Error  Cleanup VIC Appliance On Test Server
```

Comment:
We should not ignore an error here unless we are consistently expecting it to fail because we've put the VCH into a state deliberately where cleanup will fail.

I'll need more detail on why it failed before we squash all errors here.

Reply:
As discussed in person, this is necessary because (a) the state of the test suite on `releases/1.4.1` differs from `master`, (b) we want to cherry-pick a subset of relevant changes to `releases/1.4.1`, and (c) we don't want to submit changes like this directly to the release branch.

#7880 tracks re-introducing the "Get Empty VCH List When No VCH deployed" test and I've added a comment to it indicating the `Teardown` keyword should be revisited, with a pointer back to this PR.